src/meta/meta_turtle.c

```diff
@@ -8,6 +8,18 @@
 
 #include "wt_internal.h"
 
+/*
+ * Structure to hold state for metadata entry worker procedure.
+ */
+typedef struct {
+    WT_BACKUPHASH *backuphash; /* queue of target URI entries */
```

Comment:
I would add that this is only used for partial backup and that the structure is used only when we perform backup.